SimplenoteUITests/NoteList.swift

```diff
@@ -212,14 +224,27 @@ class NoteListAssert {
         NoteListAssert.noteListShown(forSelection: UID.NavBar.trash)
     }
 
+    class func noteContentIsShown(_ note: NoteData) {
+        noteContentIsShownInSearch(noteName: note.name, expectedContent: note.content)
+    }
+
     class func noteContentIsShownInSearch(noteName: String, expectedContent: String) {
-        print(">>> Asserting that note '\(noteName)' shows the following content:")
-        print(">>>> " + expectedContent)
+        print(">>> Asserting that note '\(noteName)' has the following content:")
+        print(">>>> \"\(expectedContent)\"")
+
+        guard NoteList.isNotePresent(noteName) else {
+            XCTFail(">>>> Note not found")
+            return
+        }
 
         if let noteContent = Table.getContentOfCell(noteName: noteName) {
-            XCTAssertTrue(noteContent.contains(expectedContent), "Content NOT found")
+            XCTAssertTrue(noteContent.contains(expectedContent), "Content is different.")
+        } else if expectedContent.isEmpty {
+            /// If note content is nil, but we assert for empty content, we should not fail
+            XCTAssert(true, "Content is not empty.")
         } else {
-            XCTFail("Could not find note")
+            /// Otherwise, we should fail
+            XCTFail("Note has no content. Only title.")
```

Comment:
Firstly check if the note exists, then assert the content. Should not fail in case when content is `nil` while we expect no content.